### PR TITLE
[INLONG-544][Sort] Modify doc for JDBC support ingest all changelog data

### LINKED ISSUE #544: [Feature][Sort] Modify doc for JDBC support ingests all changelog data
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by thesumery**
### Prepare a Pull Request

*[INLONG-544][Sort] Modify doc for jdbc support ingest all changelog data*
- Fixes #544

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/clickhouse.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ TODO: 将在未来支持此功能。
 | sink.buffer-flush.interval | 可选 | 1s | Duration | flush 间隔时间，超过该时间后异步线程将 flush 数据。可以设置为 '0' 来禁用它。注意, 为了完全异步地处理缓存的 flush 事件，可以将 'sink.buffer-flush.max-rows' 设置为 '0' 并配置适当的 flush 时间间隔。 |
 | sink.max-retries | 可选 | 3 | Integer | 写入记录到数据库失败后的最大重试次数。 |
 | sink.parallelism | 可选 | (none) | Integer | 用于定义 JDBC sink 算子的并行度。默认情况下，并行度是由框架决定：使用与上游链式算子相同的并行度。 |
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有类型的RowKind，全部当做INSERT摄入进来。 |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/clickhouse.md:100]: Maybe this is better?
```
忽略所有 RowKind 类型的变更日志，将它们当作 INSERT 的数据来采集。
```

**Review by EMsnap — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/oracle.md [hunk 2]
```diff
@@ -94,7 +94,12 @@ TODO: 将在未来支持此功能。
 | sink.buffer-flush.interval | 可选 | 1s | Duration | flush 间隔时间，超过该时间后异步线程将 flush 数据。可以设置为 '0' 来禁用它。注意, 为了完全异步地处理缓存的 flush 事件，可以将 'sink.buffer-flush.max-rows' 设置为 '0' 并配置适当的 flush 时间间隔。 |
 | sink.max-retries | 可选 | 3 | Integer | 写入记录到数据库失败后的最大重试次数。 |
 | sink.parallelism | 可选 | (none) | Integer | 用于定义 JDBC sink 算子的并行度。默认情况下，并行度是由框架决定：使用与上游链式算子相同的并行度。 |
+<<<<<<< HEAD
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有类型的RowKind，全部当做INSERT摄入进来。 |
 | inlong.metric | 可选 | (none) | String | inlong metric 的标签值，该值的构成为groupId&streamId&nodeId。|
+=======
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/oracle.md:100]: please fix the conflicts.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/postgresql.md [hunk 3]
```diff
@@ -95,7 +95,12 @@ TODO: 将在未来支持此功能。
 | sink.buffer-flush.interval | 可选 | 1s | Duration | flush 间隔时间，超过该时间后异步线程将 flush 数据。可以设置为 '0' 来禁用它。注意, 为了完全异步地处理缓存的 flush 事件，可以将 'sink.buffer-flush.max-rows' 设置为 '0' 并配置适当的 flush 时间间隔。 |
 | sink.max-retries | 可选 | 3 | Integer | 写入记录到数据库失败后的最大重试次数。 |
 | sink.parallelism | 可选 | (none) | Integer | 用于定义 JDBC sink 算子的并行度。默认情况下，并行度是由框架决定：使用与上游链式算子相同的并行度。 |
+<<<<<<< HEAD
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有类型的RowKind，全部当做INSERT摄入进来。 |
 | inlong.metric | 可选 | (none) | String | inlong metric 的标签值，该值的构成为groupId&streamId&nodeId。|
+=======
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有 RowKind 类型的变更日志，将它们当作 INSERT 的数据来采集。 |
+>>>>>>> ba4a76f570 ([INLONG-544][Sort] Optimize translation)
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/postgresql.md:103]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/tdsql-postgresql.md [hunk 4]
```diff
@@ -93,7 +93,12 @@ TODO: 将在未来支持此功能。
 | sink.buffer-flush.interval | 可选 | 1s | Duration | flush 间隔时间，超过该时间后异步线程将 flush 数据。可以设置为 '0' 来禁用它。注意, 为了完全异步地处理缓存的 flush 事件，可以将 'sink.buffer-flush.max-rows' 设置为 '0' 并配置适当的 flush 时间间隔。 |
 | sink.max-retries | 可选 | 3 | Integer | 写入记录到数据库失败后的最大重试次数。 |
 | sink.parallelism | 可选 | (none) | Integer | 用于定义 JDBC sink 算子的并行度。默认情况下，并行度是由框架决定：使用与上游链式算子相同的并行度。 |
+<<<<<<< HEAD
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有类型的RowKind，全部当做INSERT摄入进来。 |
 | inlong.metric | 可选 | (none) | String | inlong metric 的标签值，该值的构成为groupId&streamId&nodeId。|
+=======
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有 RowKind 类型的变更日志，将它们当作 INSERT 的数据来采集。 |
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/tdsql-postgresql.md:100]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/sqlserver.md [hunk 5]
```diff
@@ -93,7 +93,12 @@ TODO: 将在未来支持此功能。
 | sink.buffer-flush.interval | 可选 | 1s | Duration | flush 间隔时间，超过该时间后异步线程将 flush 数据。可以设置为 '0' 来禁用它。注意, 为了完全异步地处理缓存的 flush 事件，可以将 'sink.buffer-flush.max-rows' 设置为 '0' 并配置适当的 flush 时间间隔。 |
 | sink.max-retries | 可选 | 3 | Integer | 写入记录到数据库失败后的最大重试次数。 |
 | sink.parallelism | 可选 | (none) | Integer | 用于定义 JDBC sink 算子的并行度。默认情况下，并行度是由框架决定：使用与上游链式算子相同的并行度。 |
+<<<<<<< HEAD
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有类型的RowKind，全部当做INSERT摄入进来。 |
 | inlong.metric | 可选 | (none) | String | inlong metric 的标签值，该值的构成为groupId&streamId&nodeId。|
+=======
+| sink.ignore.changelog | 可选 | false | Boolean |  忽略所有 RowKind 类型的变更日志，将它们当作 INSERT 的数据来采集。 |
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/sqlserver.md:100]: ditto

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**